### ISSUE TITLE
[Dynamo] torch.manual_seed ignored

### ISSUE THREAD

**Issue by eellison** [pytorch/pytorch#107187]
### 🐛 Describe the bug

```
import torch
import torch._inductor.config
torch._inductor.config.fallback_random = True

@torch.compile()
def foo():
    torch.manual_seed(3)
    return torch.rand([4])

print(foo())
print(foo())
```

> tensor([0.2373, 0.2485, 0.7256, 0.0577])
> tensor([0.0030, 0.8033, 0.8953, 0.7815])

### Versions

master

**Comment by anijain2305** [comment]
The issue here is that `manual_seed` is invisible to AOT dispatcher, so AOT Autograd graph does not have this. There could be complicated solutions here, but a simple one is to graph break on `manual_seed`

A simpler, little faster, repro is

~~~
import torch

@torch.compile(backend="aot_eager")
def foo():
    torch.manual_seed(3)
    return torch.rand([4])

print(foo())
print(foo())
~~~

**Comment by ysiraichi** [comment]
I tried graph breaking by adding a call to `unimplemented("torch.manual_seed not supported")` in _torch/_dynamo/variables/torch.py_. Problem is: dynamo does break the graph, but it starts symbolically tracing the `torch.manual_seed` function (which, I guess, should be skipped).

I tried calling `unimplemented` early in `convert_frame_assert` function. It kind of works, but it starts tracing through a inner call for `_seed_custom_device`.

Any thoughts on how to avoid tracing through those functions?

**Comment by anijain2305** [comment]
We can use `torch._disable_dynamo` decorator on the `manual_seed` itself.

**Comment by wanchaol** [comment]
@anijain2305 curious if we would lower this manual seed op to the prim op you added in the future? this would allow aot dispatcher to capture the seed settings iiuc
